academy/4-my-own-chain/cosmjs-messages.md

```diff
@@ -0,0 +1,391 @@
+---
+title: CosmJs - Create the messages for your GUI
+order: 23
+description: Create the messages that your GUI will use
+tag: deep-dive
+---
+
+# CosmJs - Create the messages for your GUI
+
+<HighlightBox type="synopsis">
+
+Make sure you have all you need before proceeding:
+
+* You understand the concepts of [CosmJs](TODO).
```

Comment:
Broken link

Reply:
Ah yes, we will have to fix those when the files are in their ~forever home~ final folder @coldice.